projects/kubernetes-sigs/image-builder/build/setup_packer_configs.sh

```diff
@@ -74,10 +81,10 @@ export KUBERNETES_FULL_VERSION="$KUBERNETES_VERSION-eks-$RELEASE_BRANCH-$EKSD_RE
 export ETCD_HTTP_SOURCE=$(build::eksd_releases::get_eksd_component_url "etcd" $RELEASE_BRANCH)
 export ETCD_VERSION=$(build::eksd_releases::get_eksd_component_version "etcd" $RELEASE_BRANCH)
 export ETCD_SHA256=$(build::eksd_releases::get_eksd_component_sha "etcd" $RELEASE_BRANCH)
-export ETCDADM_HTTP_SOURCE=${ETCDADM_HTTP_SOURCE:-$(build::common::get_latest_eksa_asset_url $ARTIFACTS_BUCKET 'kubernetes-sigs/etcdadm' 'amd64' $LATEST_TAG)}
+export ETCDADM_HTTP_SOURCE=${ETCDADM_HTTP_SOURCE:-$(build::eksa_releases::get_eksa_component_asset_url 'eksD' 'etcdadm' $RELEASE_BRANCH $DEV_RELEASE $GIT_BRANCH)}
```

Comment:
where does eksD come from?

Reply:
These are artifacts that are under eksD on bundles for the fact we need it in the node images. These were actually under eksD/ova/ubuntu/. We recently moved it to be under eksD to make it same across eksD-level image-formats.

https://github.com/aws/eks-anywhere/blob/b4a6b6a226ef668c742751ad52f7eace4672da75/release/api/v1alpha1/bundle_types.go#L87